api/dbadapters/bigquery.ts

```diff
@@ -256,6 +258,61 @@ export class BigQueryDbAdapter implements IDbAdapter {
     await this.runQuery(updateQuery);
   }
 
+  public async setMetadata(action: dataform.IExecutionAction): Promise<void> {
+    const { target, actionDescriptor } = action;
+
+    await this.pool
```

Comment:
we shouldn't run this code for any action that isn't of type `table`, right?

Reply:
So I added an integration test for `view` type and it seems to work. Should I just run the code for actions of type `view || table`?

Reply:
yes please